packages/fxa-settings/src/components/PageDisplayName/index.test.tsx

```diff
@@ -24,3 +28,40 @@ it('renders', async () => {
   expect(screen.getByTestId('input-field')).toBeInTheDocument();
   expect(screen.getByTestId('submit-display-name')).toBeInTheDocument();
 });
+
+it('updates the disabled state of the save button', async () => {
+  renderWithRouter(
+    <AuthContext.Provider value={{ auth: client }}>
+      <MockedCache>
+        <PageDisplayName />
+      </MockedCache>
+    </AuthContext.Provider>
+  );
+
+  // initial value
+  expect(screen.getByTestId('submit-display-name')).toBeDisabled();
+
+  // empty value
+  await act(async () => {
+    fireEvent.change(screen.getAllByTestId('input-field')[0], {
```

Comment:
I needed to change `fireEvent.change` to `fireEvent.input` on the change password form. _If_ that works here I think we should prefer `input` for form fields in general.